backend/src/contracts/signer.rs

```diff
@@ -34,7 +36,10 @@ impl SummaSigner {
     ) -> Self {
         let wallet: LocalWallet = LocalWallet::from_str(main_signer_key).unwrap();
 
-        let provider = Provider::<Http>::try_from(rpc_url).unwrap();
+        // let provider = Provider::<Http>::try_from(rpc_url).unwrap();
```

Comment:
Looks like the comment is unnecessary

Reply:
Removed at [70ad9c9](https://github.com/summa-dev/summa-solvency/pull/116/commits/70ad9c97f31c3573941a23cb0ff7a36952f95392)